### DISCUSSION TITLE
how to connect nxt ?

### DISCUSSION THREAD

**Discussion by mting806** [pybricks/support#1868] [General]
hello 
This is an amazing tool for making different bricks work under same control env.
In the code.pybricks.com firmware section, it said nxt is working.
after activating the lifetime license, I still could not see nxt choice.
is there any special procedure for that? 
thank you very much

**Comment by laurensvalk** [comment]
The NXT is not supported. It does say that previous generations of Mindstorms are not supported, but I agree that we can improve the text to be clearer.

In any case, I'll be happy to provide a full refund 😊

That said, we have done some experiments with the NXT to show that it could work. Hopefully we'll be able to fully support Pybricks on the NXT in the future.

**Comment by mting806** [comment]
no need refund.
i am happy to do some support for this good project :)
I will try pybricksdev , I have seen some discussion about USB local usage etc, I could load the firmware from artfactory to the nxt.
thank you very much

**Reply by laurensvalk** [reply]
Thank you, that is very kind. Indeed, this helps our mission to support NXT in the long run! 

There isn't all that much implemented for the NXT yet, so unless you are a firmware developer yourself, don't count much in it :smile:

In the mean time, perhaps consider a tool like NXC using BrixCC.
